managedbeans/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Jakarta Managed Beans"
+summary: "Jakarta Managed Beans defines a set of basic services for container-managed objects with minimal requirements, otherwise known under the acronym "POJOs" (Plain Old Java Objects)."
```

Comment:
The nested double quotes are probably going to be a problem.
Perhaps convert the inner quotes to single quotes?
Or figure out how to escape them with backslash?
Sorry, I have no idea what hugo requires.

Reply:
I did some searching and I couldn't figure out how hugo was going to respond either.  So, I'm taking the safe way out and just removed the embedded quotes.  The acronym POJO is common enough that it doesn't have to be in quotes.  I changed it here and in the release _index.md just to be consistent.  Thanks.